resources/views/foods/index.blade.php

```diff
@@ -0,0 +1,61 @@
+@extends('layouts.master')
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">{{ trans('foods/listfoods.title') }}</h3>
+            <a href="{{ route('foods.create') }}" class="btn btn-primary btn-xl pull-right fa fa-plus"> {{ trans('foods/listfoods.add-food') }}</a>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div class="dataTables_wrapper form-inline dt-bootstrap">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table class="table table-bordered table-striped dataTable table-hover"
+                               role="grid">
+                            <thead>
+                            <tr role="row">
+                                <th class="col-md-1">{{ trans('foods/listfoods.id') }}</th>
+                                <th class="col-md-2">{{ trans('foods/listfoods.name') }}</th>
+                                <th class="col-md-2">{{ trans('foods/listfoods.category') }}</th>
+                                <th class="col-md-1">{{ trans('foods/listfoods.price') }}</th>
+                                <th class="col-md-4">{{ trans('foods/listfoods.description') }}</th>
+                                <th class="col-md-2">{{ trans('foods/listfoods.action') }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @php ($i = 1)
+                            @foreach($foods as $food)
+                                <tr>
+                                    <td>{{ $i++ }}</td>
+                                    <td>{{ $food->name }}</td>
+                                    <td>{{ $food->categories->name }}</td>
```

Comment:
-you have to use with function to get category  when you get list food in controller.